skyportal/handlers/api/source.py

```diff
@@ -218,6 +218,18 @@ def get(self, obj_id=None):
             schema:
               type: string
             description: Simbad class to filter on
+          - in: query
+            name: alias
+            nullable: true
+            schema:
+              type: string
```

Comment:
I think this should be an array of strings. There could be multiple names for the same object. Could you please update the logic where necessary?